src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+type Props = {};
+
+export const TodoHeader: React.FC<Props> = props => {
+  const {} = props;
+
+  return (
+    <header className="todoapp__header">
+      {/* this button should have `active` class only if all todos are completed */}
+      <button
+        type="button"
+        className="todoapp__toggle-all active"
+        data-cy="ToggleAllButton"
+      />
```

Comment:
The 'Toggle All' button should have the 'active' class only if all todos are completed. Currently, it is always set to 'active'. Consider adding logic to conditionally apply the 'active' class based on the completion status of all todos.